docs/contributing.rst

```diff
@@ -112,6 +112,19 @@ Documentation is written in `Sphinx-flavored reStructuredText <sphinx:index>`, s
 
 Feel free to file issues or pull requests as well if you wish something was better documented, as this too can help prioritize.
 
+Building the Documentation Locally
```

Comment:
I think this seems like it can go combined in the previous section rather than with its own heading maybe.

As in:

```
Documentation is written in `Sphinx-flavored reStructuredText <sphinx:index>`, so you'll want to at least have a quick look at the `tutorial <sphinx:tutorial>` if you have never written documentation using it.

The code for the documentation is in the ``docs/`` directory of our :org:`github repository <bowtie>`.

etc.
```

Perhaps test out combining them. Will leave another comment or two about the specific content.